CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ string( TOLOWER "${CMAKE_BUILD_TYPE}" CMAKE_BUILD_TYPE )
 
 ########
 # Compiler/linker options based on the different compilers
+ADD_DEFINITIONS(-std=c++11 )
```

Comment:
Do you really need this here? -std=c++11 is added to CXX_FLAGS just a few lines below. If you really need this: which compiler/cmake are you using?